### DISCUSSION TITLE
Scrolls to the click position

### DISCUSSION THREAD

**Discussion by Mephisto090** [cefsharp/CefSharp.Dom#19] [Q&A] [answered]
Hello,

CefSharp.Puppeteer scrolls the window to the click position to make it more realistic. Is there a function where I can do this myself without clicking

Best regards

**Comment by amaitland** [comment, marked as answer]
The method used is currently private. You are welcome to submit a PR that makes it public and adds the relevant xml doc. 

https://github.com/cefsharp/Puppeteer/blob/main/lib/PuppeteerSharp/ElementHandle.cs#L732

Or just use JavaScript

**Reply by Mephisto090** [reply]
I would like to do it and have already connected the git to my visual studio. But I can't get the code on GitHub. How does this work?

**Reply by Mephisto090** [reply]
i hope its works?

https://github.com/hardkoded/puppeteer-sharp/pull/1935

**Reply by amaitland** [reply]
Your PR is targeted at the puppeteer Sharp repository.

**Reply by Mephisto090** [reply]
As soon as I press Pull Request on cefsharp/Puppeteer and want to create a PR, I'm automatically taken to the page https://github.com/hardkoded/puppeteer-sharp/pull/1935

**Reply by amaitland** [reply]
Doesn't look like you have a public fork.

**Reply by Mephisto090** [reply]
Now?
If it worked, can you possibly write the summary? Your English is better than mine :)
